pkg/hub/manager.go

```diff
@@ -37,8 +38,26 @@ import (
 
 var ResyncInterval = 5 * time.Minute
 
+// HubManagerOptions holds configuration for hub manager controller
+type HubManagerOptions struct {
+	ClusterAutoApprovalUsers []string
+}
+
+// NewHubManagerOptions returns a HubManagerOptions
+func NewHubManagerOptions() *HubManagerOptions {
+	return &HubManagerOptions{}
+}
+
+// AddFlags registers flags for manager
+func (m *HubManagerOptions) AddFlags(fs *pflag.FlagSet) {
+	features.DefaultHubMutableFeatureGate.AddFlag(fs)
+	fs.StringArrayVar(&m.ClusterAutoApprovalUsers, "--cluster-auto-approval-users", m.ClusterAutoApprovalUsers,
```

Comment:
should remove `--` here, otherwise the flag will be:
```bash
Flags:
      ----cluster-auto-approval-users stringArray   A list of reachable spoke cluster api server URLs for hub cluster.
```